Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -15,6 +15,11 @@ namespace Microsoft.MixedReality.Toolkit.SDK.UX.Pointers
     [RequireComponent(typeof(DistorterGravity))]
     public class LinePointer : BaseControllerPointer
     {
+        [Range(2, 50)]
```

Comment:
Should we go ahead and start this at `1`, or is a minimum of 2 required? (thinking of cases like a straight line)

Reply:
also, is 50 still too many?

Reply:
In 95% of cases probably yes. There are cases where 50 could be useful - eg a really weird or inventive spline pointer. The inspector does give you a warning if you set it above 25 (soon to be 20).